tidb-cloud/vector-search-changelogs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Vector Search Changelogs
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes for the TiDB vector search feature.
+---
+
+# Vector Search Changelogs
+
+## June 24, 2024
```

Comment:
```suggestion
## June 25, 2024
```

Reply:
to be consistent with the date in https://github.com/pingcap/docs/pull/18003/files